static/css/style.css

```diff
@@ -25,6 +25,25 @@ footer {
 	border-top: 1px solid #CCCCCC;
 	display: block;
 	background-color: #402924;
+	color: #FFF;
+}
+footer img.codecatz-logo {
+	height: 38px;
+    width: 38px;
```

Comment:
also, why are you introducing such a large image if you are only using 38px width and height? wouldn't it be more optimal to resize the image and then there would be no need for this rule at all?